itests/kit/ensemble.go

```diff
@@ -340,7 +340,6 @@ func (n *Ensemble) Start() *Ensemble {
 			node.FullAPI(&full.FullNode, node.Lite(full.options.lite)),
 			node.Base(),
 			node.Repo(r),
-			node.MockHost(n.mn),
```

Comment:
I'm not sure what the impact of using a real host for all tests is...if necessary, we can have an enablep2p option just for the net tests.